vars/Fedora-24.yml

```diff
@@ -0,0 +1,5 @@
+---
+# Put internal variables here with Fedora 24 specific values.
+
+# Example:
+_template_distrocode: "CentOS 24"
```

Comment:
This looks wrong, and moreover given how fast are Fedora releases getting out of date, I think it would be preferable to just use Fedora in the file name.